components/shared/Footer.js

```diff
@@ -57,10 +51,10 @@ export default function Footer({ links }) {
                   Data & Technology Services
                 </span>
                 <span className="fs-6 fw-light py-1">
-                  Austin Transportation
+                  Austin Transportation & Public Works
```

Comment:
Would you mind using `&amp;` for the & in the copy? `&` gets a lot of special meanings in HTTP and HTML, and the other way is a HTML character entity for the ampersand.

Reply:
Got it! I'll make the change in other places as well.